modules/installation-configuration-parameters.adoc

```diff
@@ -1699,6 +1699,52 @@ Optional VMware vSphere machine pool configuration parameters are described in t
 |Integer
 |====
 
+
+.Technology preview only VMware vSphere parameters
+[cols=".^2,.^3a,.^3a",options="header"]
+|====
+|Parameter|Description|Values
+
+|`platform.vsphere.failureDomains`
+|Establishes the relationships between a region and zone. You define a failure domain by using vCenter objects, such as a `datastore` object. A failure domain defines the vCenter location for {product-title} cluster nodes.
+|String
+
+|`platform.vsphere.failureDomains.name`
+|The name of the failure domain. The machine pools use this name to reference the failure domain.
+|String
+
+|`platform.vsphere.failureDomains.region`
+|You define a region by using a tag from the `openshift-region` tag category. The tag must be attached to the vCenter datacenter.
+|String
+
+|`platform.vsphere.failureDomains.zone`
+|You define a zone by using a tag from the `openshift-zone` tag category. The tag must be attached to the vCenter datacenter.
+|String
+
+|`platform.vsphere.failureDomains.topology.computeCluster`
+|This parameter defines the compute cluster associated with the failure domain. If you do not define this parameter in your configuration, the compute cluster takes the value of `platform.vsphere.cluster` and `platform.vsphere.datacenter`.
+|String
+
+|`platform.vsphere.failureDomains.topology.folder`
+|The absolute path of an existing folder where the installation program creates the virtual machines. If you do not define this parameter in your configuration, the folder takes the value of `platform.vsphere.folder`.
+|String
+
+|`platform.vsphere.failureDomains.topology.datacenter`
+|Defines the datacenter where {product-title} virtual machines (VMs) operate. If you do not define this parameter in your configuration, the datacenter defaults to `platform.vsphere.datacenter`.
+|String
+
+|`platform.vsphere.failureDomains.topology.datastore`
+| The name of the datastore to use for provisioning volumes. If you do not define this parameter in your configuration, the datastore takes the value of `platform.vsphere.defaultDatastore`.
+|String
+
+//Richard: mention port groups? There was a request to remove such eferences in 4.13 docs. "The port groups associated with the failure domain. If not defined, the port group takes the value of `platform.vsphere.network`. Note: Only a single port group may be defined at this time."
+|`platform.vsphere.failureDomains.topology.networks`
```

Comment:
@rvanderp3 : //Richard: mention port groups? There was a request to remove such eferences in 4.13 docs. "The port groups associated with the failure domain. If not defined, the port group takes the value of `platform.vsphere.network`. Note: Only a single port group may be defined at this time."

Reply:
its fine to remove mention of port groups

Reply:
Thanks, Richard.